lib/generators/test_unit/scaffold/scaffold_generator.rb

```diff
@@ -0,0 +1,19 @@
+require "rails/generators/test_unit/scaffold/scaffold_generator"
+
+module TestUnit # :nodoc:
+  module Generators # :nodoc:
+    class ScaffoldGenerator < Base # :nodoc:
+      def fix_system_test
+        if turbo_defined?
+          gsub_file File.join("test/system", class_path, "#{file_name.pluralize}_test.rb"), "click_on \"Destroy this #{human_name.downcase}\", match: :first", "accept_confirm { click_on \"Destroy this #{human_name.downcase}\", match: :first }"
```

Comment:
Can we make this a bit less brittle (in the case where the exact text of the system test changes) by using a more permissive regular expression? Maybe something like

```suggestion
          gsub_file File.join("test/system", class_path, "#{file_name.pluralize}_test.rb"), /(click_on.*Destroy this.*)$/", "accept_confirm { \\1 }"
```

Reply:
Actually I tested this change and it works, so I added a commit doing it!

Reply:
Much nicer! thanks!